### DISCUSSION TITLE
2007/10/09/%E4%B9%B0%E8%BD%A6%E7%9A%84%E7%BB%8F%E5%8E%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#180] [Announcements]
# 2007/10/09/%E4%B9%B0%E8%BD%A6%E7%9A%84%E7%BB%8F%E5%8E%86

自打公司搬到交通极为不便的张江集电港，我就打算买辆车。但是后来，先买了辆自行车，又由于工作比较忙，买汽车的计划就被搁置了起来。买车和买房比，还是买房更实用些，一般还是应该考虑先买房的。

https://blog.qizhen.xyz/2007/10/09/%E4%B9%B0%E8%BD%A6%E7%9A%84%E7%BB%8F%E5%8E%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

xiaobai
2008年03月26日 19:49 编辑
天啊，按照你的描述、你的收入和你的购物水平，我真的不敢到上海来了，太恐怖了！

回复
Qizhen
2007年10月18日 09:04 编辑
to 辉，以后车要有什么问题，就问你了 :)

回复
辉
2007年10月17日 21:50 编辑
我是比亚迪的，呵呵
没想到阮大侠会买我们公司的车

回复
Elaine
2007年10月13日 18:23 编辑
打从自学labview开始的那一天就开始关注你的blog，也有几个月任光了吧，你真是你的神奇的人。希望一直可以关注下去！！！

回复
杰
2007年10月10日 15:49 编辑
我也想买这车的.呵呵.
适合代步用.

回复
Lu
2007年10月10日 10:47 编辑
恭喜恭喜，买车了啊！
下次去蹭车，嘻嘻…

回复
Qizhen
2007年10月10日 08:21 编辑
接你没问题，收费肯定比出租便宜

回复
奕瑋
2007年10月9日 18:50 编辑
首先恭喜阮主成为有车一族！
七八万的车，您就先凑合着开吧，总体说来，
这个价位的车比自行车强点儿有限，都或多或少有不如意的地方。
就算你将来买几十万的车开，也是会有不满的。
关键是它要适合你现在的经济基础和使用需求。
其实我现在开着夏利N3跑京津塘高速也没感觉太不爽。
另外，用这个车先把驾驶经验积累起来，将来就好办了。
下次我去上海，你能接我吗？哈哈！
